app/views/students/_left_nav_show.html.haml

```diff
@@ -1,6 +1,6 @@
-%ul{ class: 'side-nav'}
+%ul{ class: 'vertical menu'}
   %li.heading Planning
-  %li.calendar= link_to "Calendar", "#"
-  %li.courses= link_to "Subjects", "#"
-  %li.books= link_to "Resources", "#"
-  %li.delete_student= link_to "Delete Student", @student, method: :delete
+  %div{ class: "calendar link" }= link_to "Calendar", "#"
+  %div{ class: "courses link" }= link_to "Subjects", "#"
+  %div{ class: "books link" }= link_to "Curriculum", "#"
+  %div{ class: "delete_student link" }= link_to "Delete Student", @student, method: :delete
```

Comment:
Line is too long. [91/80]<br>Avoid defining `class` in attributes hash for static class names